develop/components/pages/TextAreaPage.tsx

```diff
@@ -47,8 +48,8 @@ export default class TextAreaPage extends React.Component<any, any>  {
                                 error="error msg will be shown here"
                                 cols={30}
                                 rows={5}
-                                value={this.state.textBoxValue}
-                                onChange={(event) => { this.setState({ textBoxValue: event.target.value }); }}
+                                value={this.state.textBox2Value}
+                                onChange={(event) => { this.setState({ textBox2Value: event.target.value }); }}
```

Comment:
Not a pressing issue, but try and avoid arrow functions within react ```render``` lifecyle as much as possible. So 

`const textChange  = (event) => { this.setState({ textBox2Value: event.target.value }); }

and 

 onChange={this.textChange}
`

Reply:
Thanks @DanSallau I'll keep that in mind in the future. For now I didn't want to refactor the doc pages too much as I would like to refactor every page in the future as part of a general refactor. Maybe you can open an issue for it. 

*Side note: AFAIK this perf issue only affects older browsers right? :thinking: :*

Reply:
Nope @mario-subo its not browser based but rather react-engine issue. Arrow function doesn't have a `this` of its own , it takes the parent's own and in this case the component. Also `() => ` returns a new function all the time and that is considered a state change by react engine.

Reply:
Ok thanks for that. I will leave it in for now and refactor on a separate large refactor branch soon